paper-toast.html

```diff
@@ -48,6 +48,7 @@
 ----------------|-------------|----------
 `--paper-toast-background-color` | The paper-toast background-color | `#323232`
 `--paper-toast-color` | The paper-toast color | `#f1f1f1`
+`--paper-font-common-base` | Mixin applied to the toast for setting the font, defined in `paper-styles/typography.html` | `{}`
```

Comment:
In that case, the default value isn't `{}`, it's "the one defined in `paper-styles/typography.html"